site/content/docs/01-component-format.md

```diff
@@ -130,6 +130,24 @@ Any top-level statement (i.e. not inside a block or a function) can be made reac
 
 ---
 
+Only values which directly appear within the `$:` block will become dependencies of the reactive statement. For example, in the code below `total` will only update when `x` or `y` change, but not `z`.
```

Comment:
This PR seems unnecessary to me because I don't really understand why anyone would think `z` would update. There's nothing that sets `z`'s value. Independent of reactivity concepts, it seems like `z` will always be `0`. Am I missing something?

Reply:
Nothing would prevent Svelte from parsing the definition of updateTotal, discovering that z is referenced, and then make the total expression also reactive on z.

Reply:
@benmccann oh but yeah in this component z is always 0, but even if you add code to change it, changing z in and of itself does not trigger a re-calculation of total.

Reply:
Oh, I see the point you're making now. A couple suggestions:
- `x` and `y` in `updateTotal` shadow the `x` and `y` from the outer scope. I think it'd be clearer to use another variable name in this `function`
- we don't really need both `x` and `y` to demonstrate this. a single variable would be enough
- it might be helpful to add another line that changes `x` and one that changes `z` and show what `total` is after each of them

Reply:
sure

Reply:
@benmccann good feedback, thanks. How's this look?

Reply:
thanks for the changes!